src/test/java/org/jabref/logic/exporter/XmpPdfExporterTest.java

```diff
@@ -61,46 +61,46 @@ class XmpPdfExporterTest {
     private FilePreferences filePreferences;
 
     private static void initBibEntries() throws IOException {
-        olly2018.setCitationKey("Olly2018");
-        olly2018.setField(StandardField.AUTHOR, "Olly and Johannes");
-        olly2018.setField(StandardField.TITLE, "Stefan's palace");
-        olly2018.setField(StandardField.JOURNAL, "Test Journal");
-        olly2018.setField(StandardField.VOLUME, "1");
-        olly2018.setField(StandardField.NUMBER, "1");
-        olly2018.setField(StandardField.PAGES, "1-2");
-        olly2018.setMonth(Month.MARCH);
-        olly2018.setField(StandardField.ISSN, "978-123-123");
-        olly2018.setField(StandardField.NOTE, "NOTE");
-        olly2018.setField(StandardField.ABSTRACT, "ABSTRACT");
-        olly2018.setField(StandardField.COMMENT, "COMMENT");
-        olly2018.setField(StandardField.DOI, "10/3212.3123");
-        olly2018.setField(StandardField.FILE, ":article_dublinCore.pdf:PDF");
-        olly2018.setField(StandardField.GROUPS, "NO");
-        olly2018.setField(StandardField.HOWPUBLISHED, "online");
-        olly2018.setField(StandardField.KEYWORDS, "k1, k2");
-        olly2018.setField(StandardField.OWNER, "me");
-        olly2018.setField(StandardField.REVIEW, "review");
-        olly2018.setField(StandardField.URL, "https://www.olly2018.edu");
+        OLLY_2018.setCitationKey("Olly2018");
```

Comment:
Can we do withCitationKey and move the initialization to the variable?

Reply:
Yes, but moving the initialization to the variable is a bad idea due to how the test works. Due to MethodSources it has to be static and we are setting the vars to the correct values and we are resetting them again.